### PR TITLE
Video-12-Add-Redux-To-Home-Screen

### PR DESCRIPTION
**Pull request by basir**

### CODE REVIEW

**Comment by TafadzwaTaks** [comment]
Hello basir hope i find you well , i  am having problems with adding redux to HomeScreen. after successfully compiling with the npm start function my browser is showing me this error (TypeError: Object(...) is not a function)
This is where the code is showing errors i have uploaded images so it can be clearer . 
error message 2
error message 1

**Please help me sir i have been stuck for some time now**

File: frontend/src/reducers/productReducers.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+const {
+  PRODUCT_LIST_REQUEST,
+  PRODUCT_LIST_SUCCESS,
+  PRODUCT_LIST_FAIL,
+} = require('../constants/productConstants');
```
Comment (YitshakS) [inline frontend/src/reducers/productReducers.js:5]: According to ES6 it should be:

**import** {
	PRODUCT_LIST_REQUEST,
	PRODUCT_LIST_SUCCESS,
	PRODUCT_LIST_FAIL,
} **from** "../constants/productConstants";

**Comment by Dewruwan95** [comment]
Dear Sir,
In this part, I have an issue. It shows duplicates of these actions. I can't find the Issue. Please help me.
image

File: frontend/src/store.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { createStore, compose, applyMiddleware, combineReducers } from 'redux';
```
Comment (quavo19) [inline frontend/src/store.js:1]: this method is depreciated. you have to import createStore like this 
`import { configureStore } from '@reduxjs/toolkit';` separately

**Pull request merged**